### PR TITLE
Feature/add serilog to backend

### LINKED ISSUE #9: Setup and confiugre Serilog for backend apps
Setup backend apps to use Serilog to write their logs in a file and console.
- [x] DeUrgenta.Api
- [x] DeUrgenta.RecurringJobs

### PR DESCRIPTION
**Pull request by idormenco**
closes #9

### CODE REVIEW

File: Src/DeUrgenta.RecurringJobs/Program.cs [hunk 1]
```diff
@@ -1,23 +1,50 @@
...
+using Serilog.Sinks.SystemConsole.Themes;
 
 namespace DeUrgenta.RecurringJobs
 {
     public class Program
     {
         public static void Main(string[] args)
         {
-            CreateHostBuilder(args).Build().Run();
+            const string loggerTemplate = @"{Timestamp:yyyy-MM-dd HH:mm:ss} [{Level:u4}]<{ThreadId}> [{SourceContext:l}] {Message:lj}{NewLine}{Exception}";
```
Comment (bvizureanu) [inline Src/DeUrgenta.RecurringJobs/Program.cs:15]: could this maybe be refactored and reused across projects?

File: Src/.env [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM_EMAIL=noreply@deurgenta.ro
+SENDGRID_API_KEY=
+ASPNETCORE_URLS=http://+:80
+DB_CONNECTION_STRING=Server=postgres;Port=5432;Database=de-urgenta-db;User Id=docker;Password=docker;
+TEMPLATE_FOLDER=EmailTemplates
+SWAGGER_REDIRECT_URIS=https://localhost:5040/swagger/oauth2-redirect.html
+SWAGGER_CORS=https://localhost:5040
+IDENTITYSERVER_URL=http://localhost:5042
+HANGFIRE_USERNAME=admin
+HANGFIRE_PASSWORD=demo1234
```
Comment (bvizureanu) [inline Src/.env]: not really sure we want to keep this file in repo.. 

@catileptic , @tudoramariei , feedback pls

Comment (idormenco) [reply Src/.env]: sorry ,yes I will add to docker ignore

File: Src/DeUrgenta.RecurringJobs/Program.cs (hunk 1 above)
Comment (idormenco) [reply Src/DeUrgenta.RecurringJobs/Program.cs:15]: @bvizureanu can we add an task with low priority to improve this ?

**Review by bvizureanu — APPROVED** [review]

**Pull request merged**